index.html

```diff
@@ -19,6 +19,14 @@
         <header>
             <a href="/" class="logo">My Logo</a>
             <nav class="menu" id="icon"><i class="fa fa-bars "></i></div>
+                
+                <ul class="navigations">
+            
+                    <li> <a href="" > Protfolio</a> </li>
+                    <li> <a href="" >About</a> </li>
+                    <li> <a href="" > Contacts</a> </li>
+            </ul>
+            </nav>
         </header>
         <main>
           <section class="section-1" id="Portfolio">
```

Comment:
good job on implementing desktop version of this section  👍💪 , only small things to adjust kindly try to make the header and headline section fit the first screen as in your project we need to scroll down to see the rest and this section contain only a headline and social links ,  also the background image is stretching and not well positioned please do your best to make it look like in the design   
**figma**
![figma headline](https://user-images.githubusercontent.com/59707859/230371186-5a2d159a-8b1d-468e-b989-4952560de331.PNG)

**yours**
![student headline](https://user-images.githubusercontent.com/59707859/230371434-959142a1-b4e0-4c32-89ae-ef472a76ca77.PNG)